recipes/esmp/meta.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+{% set name = "esmp" %}
+{% set version = "ESMF_6_3_0rp1_ESMP_01" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}.{{ version }}.tar.gz
+  url: http://uvcdat.llnl.gov/cdat/resources/esmp.{{ version }}.tar.bz2
+  sha256: 582427637e916f50ced7517bfc8f96847cffeaf458fdab1420af5cdee0d8e604
+
+build:
+  number: 0
+  skip: True  # [win or py3k] 
+
+requirements:
+  build:
+    - python
+    - numpy
+    - gcc 
+  run:
+    - python
+    - numpy
+    - gcc
+
+test:
+  requires:
+    - python
+    - numpy
+    - gcc
```

Comment:
Nothing that is listed in the `run` dependencies need to be here because they will be already. Only tests only dependencies.